src/memberdef.cpp

```diff
@@ -4346,9 +4350,37 @@ void MemberDefImpl::enableCallGraph(bool e)
   if (e) Doxygen::parseSourcesNeeded = TRUE;
 }
 
+void MemberDefImpl::enableCallGraph(bool e, bool e2)
+{
+  static bool callGraph = Config_getBool(CALL_GRAPH);
+  if (e != e2)
```

Comment:
I don't understand the logic here, `e2` is not used other than comparing to `e`, and if both are not equal both are ignored...

Reply:
In case the `e` and `e2` are identical both are the same as the configuration parameter or both have been changed (in the same way by either `\show...`or `\hide...`).
I case of a difference one of the two has been changed and doesn't have the value of the configuration parameter anymore, but we don't know which one, to get the right value we need the opposite of the configuration parameter.